tests/test_transcription.py

```diff
@@ -2,27 +2,30 @@
 import json
 import pytest
 import sys
+import os.path
 
 # from .conftest import option
 from deepgram import Deepgram
 from .mock_response import MOCK_RESPONSE
 
+
+CURRENT_DIRECTORY = os.path.split(os.path.abspath(__file__))[0]
+
 api_key = pytest.api_key
 assert api_key, "Pass Deepgram API key as an argument: `pytest --api-key <key> tests/`"
 
 deepgram = Deepgram(api_key)
 
-MOCK_TRANSCRIPT = "Yep. I said it before, and I'll say it again. Life moves pretty fast. You don't stop and look around once in a while. you could miss it."
+MOCK_TRANSCRIPT = "Yep. I said it before, and I'll say it again. Life moves pretty fast. You don't stop and look around once in a while. you could miss it"
```

Comment:
I set up this test strategy - maybe there is a more forgiving way than string exact match? Like having no more than X character edit distance from expected transcript. Otherwise these small transcription changes will keep making this kind of test fail. But that could also be out of scope of this PR. What do you think?

Reply:
I agree, something that allows small differences in the transcript would be nice. We can use the `fuzzywuzzy` package :) to do this. I can add it to this PR. I'll add a requirements.txt file as well since we'll have multiple dependencies for testing after this PR.